src/screens/DiscoveryScreenV2/fragment/NewsFragment.js

```diff
@@ -1,140 +1,161 @@
 import * as React from 'react';
-import { StyleSheet, Text, View } from 'react-native'
-import { useNavigation } from '@react-navigation/native';
+import {StyleSheet, Text, View} from 'react-native';
+import {useNavigation} from '@react-navigation/native';
 
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import LoadingWithoutModal from '../../../components/LoadingWithoutModal';
 import RecentSearch from '../elements/RecentSearch';
 import RenderItem from '../elements/RenderItem';
 import share from '../../../utils/share';
 import useIsReady from '../../../hooks/useIsReady';
-import { Context } from '../../../context/Store'
-import { colors } from '../../../utils/colors';
-import { fonts } from '../../../utils/fonts';
-import { getUserId } from '../../../utils/users';
-import { newsDiscoveryContentParamBuilder } from '../../../utils/navigation/paramBuilder';
-import { withInteractionsManaged } from '../../../components/WithInteractionManaged';
+import {Context} from '../../../context/Store';
+import {fonts} from '../../../utils/fonts';
+import {getUserId} from '../../../utils/users';
+import {newsDiscoveryContentParamBuilder} from '../../../utils/navigation/paramBuilder';
+import {withInteractionsManaged} from '../../../components/WithInteractionManaged';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to import `COLORS` from the `utils/theme` module instead of `colors`. This is a good practice as it ensures consistency in color usage across the application. However, make sure that all instances of `colors` have been replaced with `COLORS` throughout the codebase to avoid any undefined errors.



<!-- This is an auto-generated comment by OSS CodeRabbit -->